ui/components/app/confirm-page-container/confirm-page-container-header/confirm-page-container-header.component.js

```diff
@@ -64,7 +75,23 @@ export default function ConfirmPageContainerHeader({
             </span>
           </div>
         )}
-        {isFullScreen ? null : <NetworkDisplay />}
+        {isFullScreen ? null : (
+          <PickerNetwork
```

Comment:
Let's update the underlying HTML element so it's no longer rendering a button in the HTML and can't be interacted with. This should also prevent you from having to add any additional CSS 
```suggestion
          <PickerNetwork
            as="div"
```

Reply:
updated. didn't know you could cast the HTML element with this Polymorphic stuff. Cool.